public/options.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
```

Comment:
```suggestion
    <title>設定 | Manaba Report Integration by Twin:te</title>
```
細かい話なのですが、ユーザに見える部分ですのでしっかりしたページの名前をつけてあげたいです！
![image](https://user-images.githubusercontent.com/45098934/207926284-38f16257-6929-4c3d-9be8-88feba91644c.png)